modules/hcp-support-matrix.adoc

```diff
@@ -91,42 +112,47 @@ The following table indicates which {product-title} versions are supported for e
 For {ibm-power-title} and {ibm-z-title}, you must run the control plane on machine types based on 64-bit x86 architecture, and node pools on {ibm-power-title} or {ibm-z-title}.
 ====
 
-In the following table, Management cluster version refers to the {product-title} version where the {mce-short} is enabled:
+In the following table, the management cluster version is the {product-title} version where the {mce-short} is enabled:
 
 .Required {product-title} versions for platforms
 [cols="3",options="header"]
 |===
 |Hosted cluster platform |Management cluster version |Hosted cluster version
 
 |{aws-full}
-|4.16 - 4.17
-|4.16 - 4.17
+|4.16 - 4.18
+|4.16 - 4.18
 
 |{ibm-power-title}
-|4.17
-|4.17
+|4.17 - 4.18
+|4.17 - 4.18
 
 |{ibm-z-title}
-|4.17
-|4.17
+|4.17 - 4.18
+|4.17 - 4.18
 
 |{VirtProductName}
-|4.14 - 4.17
-|4.14 - 4.17
+|4.14 - 4.18
+|4.14 - 4.18
 
 |Bare metal
-|4.14 - 4.17
-|4.14 - 4.17
+|4.14 - 4.18
+|4.14 - 4.18
 
 |Non-bare-metal agent machines (Technology Preview)
-|4.16 - 4.17
-|4.16 - 4.17
+|4.16 - 4.18
+|4.16 - 4.18
 |===
 
 [id="hcp-matrix-updates_{context}"]
 == Updates of {mce-short}
 
-When you update to another version of the {mce-short}, your hosted cluster can continue to run if the HyperShift Operator that is included in the version of {mce-short} supports the hosted cluster version. The following table shows which hosted cluster versions are supported on which updated {mce-short} versions:
+When you update to another version of the {mce-short}, your hosted cluster can continue to run if the HyperShift Operator that is included in the version of {mce-short} supports the hosted cluster version. The following table shows which hosted cluster versions are supported on which updated {mce-short} versions.
+
+[NOTE]
+====
+Although the HyperShift Operator supports the versions in the following table, {mce-short} supports only as far back as 2 versions earlier than the current version. For example, if the current version is 4.18, {mce-short} supports as far back as version 4.16. If you want to use a version that is earlier than one of the versions that {mce-short} supports, you can detach your hosted clusters from {mce-short} to be unmanaged.
```

Comment:
If you want to use a version that is earlier than one of the versions that {mce-short} supports, you can detach your hosted clusters from {mce-short} to be unmanaged. OR you can use an earlier version of MCE.

Also can we add a sentence or something to tell users to refer to the MCE support matrix?